### DISCUSSION TITLE
Not receiving incoming calls

### DISCUSSION THREAD

**Discussion by Ferdinand77Raw** [flutterjanus/flutter_janus_client#114] [Q&A]
Hello janus developers, I'm using the app through a FreePBX VoIp. The app works sweet when it comes to outgoing calls. I can make calls throughout the US without problems, but the problem arises when I want to receive calls. Even though, It pops up the message  when incoming call event is triggered, when I accept the call this is hanged up automatically. I will attach the full log since the start of the debugging to the event itself and a few screen captures. 

[JanusAppLog.txt](https://github.com/flutterjanus/flutter_janus_client/files/10561391/JanusAppLog.txt)
![Screenshot_1675283169](https://user-images.githubusercontent.com/64377773/216156599-cb9f208e-11e8-4dd6-9ba8-e143bf773b0a.png)
![Screenshot_1675283123](https://user-images.githubusercontent.com/64377773/216156602-9104c91d-ac5b-4f3b-8d96-6e8620675557.png)
![Screenshot_1675283110](https://user-images.githubusercontent.com/64377773/216156605-05b75987-d4f2-462a-997c-010be3777391.png)

**Comment by shivanshtalwar** [comment]
i can investigate this if you can share your sip credentials that you are using for testing on shivanshtalwar0@gmail.com with link to this discussion, most likely it is something related to sip server.

**Reply by Ferdinand77Raw** [reply]
Cool, I'll be sending the credentials to your email.
